compiler/apps/playground/components/Editor/ConfigEditor.tsx

```diff
@@ -36,15 +42,27 @@ export default function ConfigEditor({
           display: isExpanded ? 'block' : 'none',
         }}>
         <ExpandedEditor
-          onToggle={setIsExpanded}
+          onToggle={() => {
+            startTransition(() => {
+              addTransitionType('config-panel');
```

Comment:
This type could be a constant as its important it matches across usages through these modules but should also be unique from anything used in future unrelated transitions.

You could have something like `apps/playground/lib/transitionTypes.ts` where you export from.